src/advanced/bitcoin-anchoring.md

```diff
@@ -353,33 +312,34 @@ This process can suspend the anchoring transaction for a fairly large amount of
 time. For example, if the service waits for 24 confirmations, the total pause
```

Comment:
Now there is no need for service to wait enough confirmations for any Bitcoin transactions except of the fundings.